.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
@vikhyatsharma we are using yarn for this repo. Please delete the extra files that you commited. And add the raster cache to the `api-flask/.gitignore`